dev/com.ibm.ws.security.spnego/src/com/ibm/ws/security/spnego/internal/SpnegoConfigImpl.java

```diff
@@ -162,6 +168,12 @@ protected void processConfig(Map<String, Object> props) {
      */
     protected String processKrb5Keytab(Map<String, Object> props) {
         String keytab = (String) props.get(KEY_KRB5_KEYTAB);
+        Path kerbKeytab = kerbSvc.getKeytab(); // from the <kerberos> element
+
+        if (keytab == null && kerbKeytab != null) {
```

Comment:
discussion with Ut: Change to error if both are specified to different value (same as config)